dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -258,12 +258,15 @@ private Tool refreshContainer(final long containerId, final long userId) {
     @Timed
     @UnitOfWork
     @Path("/{containerId}")
-    @ApiOperation(value = "Get a registered repo", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class)
+    @ApiOperation(value = "Get a registered repo", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class, notes = "Also loads the users")
```

Comment:
Please elaborate more helpfully
i.e. loading users also loads profile information. Most endpoints returning lists of workflows/tools don't container user information

Might want to separate into notes and description appropriately?